wasm.md

```diff
@@ -819,7 +814,7 @@ A type use should start with `'(' 'type' x:typeidx ')'` followed by a group of i
 ### Type Declaration
 
 Type could be declared explicitly and could optionally bind with an identifier.
-The `identifier` for `param` of the keyword will be used 
+The `identifier` for `param` of the keyword will be used.
```

Comment:
@QianyangPeng I think this sentence was never properly completed. Feel free to fix it.

Reply:
ah yes. It is an outdated sentence. Now the `identifier` for `param` of the keyword will be discarded because it is not useful. I have updated it.